resources/assets/helpers.js

```diff
@@ -73,6 +73,22 @@ export function displayCityState(city, state) {
   return `${city || ''}${city && state ? ', ' : ''}${state || ''}`;
 }
 
+/**
+ * Returns a readable Post text string.
+ *
+ * @param {Array} post
+ * @return {String|null} text string.
+ */
+export function displayPostText(post) {
```

Comment:
Didn't you remove this helper in your other PR? Does it turn out we still need it?

Reply:
@katiecrane yes sorry that was confusing! I thought we didn't need it but turns out we do in order for the text to display on the signup page for posts! We grab posts from the database on the signup page and via the API on inbox and campaign view pages.

Reply:
Ahhh so the format is different depending on if it comes from the API or the db?

Reply:
yes exactly! From the API, the [transformer returns it under `media`](https://github.com/DoSomething/rogue/blob/master/app/Http/Transformers/PostTransformer.php#L41) while we're just grabbing the post from the database so can do `post.text`.